### ISSUE TITLE
Unable to use when inherited

### ISSUE THREAD

**Issue by calvingit** [RxSwiftCommunity/RxViewModel#52]
Install from Cocoapods, the Xcode shows below error message:

```
'disposeBag' is inaccessible due to 'internal' protection level
```

**Comment by esttorhe** [comment]
Hey @calvingit thanks for reporting.

Would you mind telling me which version of `Xcode` and also which version of `RxViewModel` are you trying to use?

Thanks

**Comment by calvingit** [comment]
Xcode: 8.3.3
RxViewModel: 5.0.0

**Comment by freak4pc** [comment]
@calvingit Could you change 

RxViewModel.swift:L25 to 
`public private(set) var disposeBag = DisposeBag()` 

And confirm if that helps your issue ?

**Comment by calvingit** [comment]
No problem if the `disposeBag` is `public`

**Comment by beretis** [comment]
Guys, disposeBag is still not public, every-time i update pods i have to edit it to be public and I think this class is supposed to be superclass. Would you mind changing please? I can make PR buts its so tiny change that its a waste of time :)
Thanks for your time

**Comment by freak4pc** [comment]
This project isn't super active - feel free to make a PR with the code I suggested above: 
```swift
public private(set) var disposeBag = DisposeBag()
```

**Comment by beretis** [comment]
Guys, disposeBag is still not public, every-time i update pods i have to edit it to be public and I think this class is supposed to be superclass. Would you mind changing please? I can make PR buts its so tiny change that its a waste of time :)
Thanks for your time

**Comment by calvingit** [comment]
Check PR #63

**Comment by esttorhe** [comment]
Fixed with #63 and adb786526eff50f219071e0dba3fb47846b9e71e
